tests/PersistentArrayTest.php

```diff
@@ -65,7 +65,7 @@ public function testSaveAs()
 
         $m = new Male($p);
         $m->load(1);
-        $m->saveAs(new Female());
+        $m->saveAs(Female::class);
```

Comment:
You're right that it's more logical to use class name here not object itself, but this change will break a lot of old code everywhere because historically we used `add('className')` A LOT :(

Reply:
So perfect then? :)

Reply:
As I said - it looks better, but will break existing projects (again) :(
Can you try to track down @romaninsh and ask what he thinks about this?

Reply:
@DarkSide666 I reacted to this as you are writing:

> because historically we used add('className')

and this method accepts now exactly this "string className" :)

Reply:
Yes and it should be documented like so. This prevents mistakes, allow strong typing and also help the developer to understant what is really does.

Imagine `newInstance` like new, you never want to allow `new (new class())()`, it is simply wrong.